main.js

```diff
@@ -21,6 +21,8 @@ const path = require('path');
 const URL = require('url');
 const config = require('./app/features/config');
 
+const showDevTools = Boolean(process.env.SHOW_DEV_TOOLS);
```

Comment:
How about doing a command line argument too?

Reply:
That would also work on release builds for example.

Reply:
Good idea, adding it

Reply:
Add command line support